pkg/operator/ceph/cluster/watcher.go

```diff
@@ -185,9 +185,14 @@ func (c *clientCluster) handleNodeFailure(ctx context.Context, cluster *cephv1.C
 		return nil
 	}
 
-	err = c.unfenceAndDeleteNetworkFence(ctx, *node, cluster)
+	err = c.unfenceAndDeleteNetworkFence(ctx, *node, cluster, "rbd")
```

Comment:
why hardcoding can we define a variable?

Reply:
Sure, updated, Thanks.